packages/wagmi/hooks/usePairTotalSupply.ts

```diff
@@ -21,7 +20,7 @@ export const usePairTotalSupply = (pair: Pair | undefined | null, chainId: Parac
       const zlp = new Token({
         address,
         name: 'Zenlink LP Token',
-        decimals: 18,
+        decimals: chainId === ParachainId.AMPLITUDE ? 12 : 18,
```

Comment:
I think it's no need to add this, the hooks in wagmi package is about evm network